tests/utils.test.js

```diff
@@ -7,6 +7,7 @@ import {
   parseBoolean,
```

Comment:
It will be awesome if you can add test of this feature in [renderStatsCard.test.js](https://github.com/anuraghazra/github-readme-stats/blob/master/tests/renderStatsCard.test.js)